### PR TITLE
Update android-corporate-owned-work-profile-enroll.md

### PR DESCRIPTION
**Pull request by chmaguir**

### CODE REVIEW

File: memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md [hunk 1]
```diff
@@ -75,6 +75,36 @@ You must create an enrollment profile so that users can enroll corporate-owned w
...
+1. Choose **Devices** > **Android** > **Android enrollment** > **Android Enterprise** > **Corporate-owned devices with work profile**.
+2. From the list, select your enrollment profile. 
+2. Select **Token**. 
+
+Another way to find the token is:
+1. Choose **Devices** > **Android** > **Android enrollment** > **Android Enterprise** > **Corporate-owned devices with work profile**.
+2. Locate your profile in the list, and then select the **More** (**...**) menu that's next to it.
+3. Select **View enrollment token**.  
+
+The token appears as an 8-digit string and a QR code. Use this token to enroll based on the enrollment mechanisms described in the [Android Enterprise corporate-owned device enrollment document](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md:91]:
```suggestion
The token appears as an eight-digit string and a QR code. Use this token to enroll based on the enrollment mechanisms described in the [Android Enterprise corporate-owned device enrollment document](/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll). 
```

File: memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md [hunk 2]
```diff
@@ -75,6 +75,36 @@ You must create an enrollment profile so that users can enroll corporate-owned w
...
+3. Select **View enrollment token**.  
+
+The token appears as an 8-digit string and a QR code. Use this token to enroll based on the enrollment mechanisms described in the [Android Enterprise corporate-owned device enrollment document](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll). 
+
+### Revoke or Export tokens
+
+- **Revoke token**: You can immediately expire the token/QR code. From this point on, the token/QR code is no longer usable. You might use this option if you:
+  - accidentally share the token/QR code with an unauthorized party
+  - complete all enrollments and no longer need the token/QR code
+- **Export token**: You can export the JSON content of the token/QR code. You might use this option to easily paste JSON content to enroll with [Zero Touch Enrollment (ZTE)](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-google-zero-touch) or [Knox Mobile Enrollment (KME)](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-knox-mobile-enrollment).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md:98]:
```suggestion
- **Export token**: You can export the JSON content of the token/QR code. You might use this option to easily paste JSON content to enroll with [Zero Touch Enrollment (ZTE)](/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-google-zero-touch) or [Knox Mobile Enrollment (KME)](/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-knox-mobile-enrollment). 
```

File: memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md [hunk 3]
```diff
@@ -75,6 +75,36 @@ You must create an enrollment profile so that users can enroll corporate-owned w
...
+  - complete all enrollments and no longer need the token/QR code
+- **Export token**: You can export the JSON content of the token/QR code. You might use this option to easily paste JSON content to enroll with [Zero Touch Enrollment (ZTE)](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-google-zero-touch) or [Knox Mobile Enrollment (KME)](https://learn.microsoft.com/en-us/mem/intune/enrollment/android-dedicated-devices-fully-managed-enroll#enroll-by-using-knox-mobile-enrollment). 
+
+Revoking or exporting a token/QR code won't have any effect on devices that are already enrolled.
+
+1. Sign in to the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431) and choose **Devices** > **Android** > **Android enrollment** > **Corporate-owned devices with work profile**.
+2. Choose the profile that you want to work with.
+3. Choose **Token**.
+5. To revoke the token, choose **Revoke token** > **Yes**.
+6. 6. To export the token, choose **Export token**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/android-corporate-owned-work-profile-enroll.md:106]:
```suggestion
6. To export the token, choose **Export token**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chmaguir, kindly commit the suggested changes. Many thanks.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
@lenewsad - Please review. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**